src/Sylius/Bundle/CoreBundle/PaymentRequest/CommandProvider/Offline/StatusPaymentRequestCommandProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\PaymentRequest\CommandProvider\Offline;
+
+use Sylius\Bundle\CoreBundle\PaymentRequest\Command\Offline\StatusPaymentRequest;
+use Sylius\Bundle\CoreBundle\PaymentRequest\CommandProvider\PaymentRequestCommandProviderInterface;
+use Sylius\Component\Payment\Model\PaymentRequestInterface;
+
+final class StatusPaymentRequestCommandProvider implements PaymentRequestCommandProviderInterface
+{
+    public function supports(PaymentRequestInterface $paymentRequest): bool
+    {
+        return $paymentRequest->getType() === PaymentRequestInterface::DATA_TYPE_CAPTURE;
```

Comment:
```suggestion
        return $paymentRequest->getType() === PaymentRequestInterface::DATA_TYPE_STATUS;
```
At least that's used in the tag

Reply:
Oups copy/paste mistake here...
BTW, I have to found a way to duplicate tests to be able to test this part.
Currently only Payum gateways are tested, meaning the Offline payment method is the one from Payum, but we can change the tag priority to be able to use the non Payum Offline payment method.